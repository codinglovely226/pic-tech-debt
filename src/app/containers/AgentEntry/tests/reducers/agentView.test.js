/* eslint-disable */

import agentViewReducer from '../../reducers/agentView';

import {
  hideAgentModal,
  resetAgentView,
  showAgentModal,
} from '../../actions/case';

describe('agentViewReducer', () => {
  it('returns the initial state', () => {
    expect(agentViewReducer(undefined, {})).toMatchSnapshot();
  });
  it('hide agent modal', () => {
    expect(agentViewReducer({}, hideAgentModal())).toMatchSnapshot();
  });
  it('reset agent view', () => {
    expect(agentViewReducer({}, resetAgentView())).toMatchSnapshot();
  });
  it('show agent modal', () => {
    expect(agentViewReducer({}, showAgentModal(0))).toMatchSnapshot();
  });
});
