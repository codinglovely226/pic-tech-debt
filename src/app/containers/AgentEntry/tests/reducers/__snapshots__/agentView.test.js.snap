exports[`agentViewReducer hide agent modal 1`] = `
Object {
  "attachments": Array [],
  "content": "",
  "show": false,
}
`;

exports[`agentViewReducer reset agent view 1`] = `
Object {
  "attachments": Array [],
  "content": "",
  "show": false,
}
`;

exports[`agentViewReducer returns the initial state 1`] = `
Object {
  "attachments": Array [],
  "content": "",
  "show": false,
}
`;

exports[`agentViewReducer show agent modal 1`] = `
Object {
  "attachments": Array [],
  "content": "",
  "show": true,
}
`;
